providers/openfeature-provider-flagd/tests/e2e/flagd_container.py

```diff
@@ -0,0 +1,57 @@
+import time
+
+from testcontainers.core.container import DockerContainer
+from testcontainers.core.waiting_utils import wait_container_is_ready, wait_for_logs
+
+HEALTH_CHECK = 8014
+
+
+class FlagDContainer(DockerContainer):
+    def __init__(
+        self,
+        image: str = "ghcr.io/open-feature/flagd-testbed:v0.5.13",
+        port: int = 8013,
+        **kwargs,
+    ) -> None:
+        super().__init__(image, **kwargs)
+        self.port = port
+        self.with_exposed_ports(self.port, HEALTH_CHECK)
+
+    def start(self) -> "FlagDContainer":
+        super().start()
+        self._checker(self.get_container_host_ip(), self.get_exposed_port(HEALTH_CHECK))
+        return self
+
+    @wait_container_is_ready(ConnectionError)
+    def _checker(self, host: str, port: int) -> None:
+        # First we wait for Flagd to say it's listening
+        wait_for_logs(
+            self,
+            "listening",
+            5,
+        )
+
+        time.sleep(1)
+        # # Second we use the GRPC health check endpoint
+        # with grpc.insecure_channel(host + ":" + port) as channel:
```

Comment:
Can this be removed?

Reply:
I reenabled the health check already - I do think it is a great way to check if the container is ready. I just need to figure out, how to do this with the sync testbed. But that is anyways a follow up